<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addemployeedialog</class>
 <widget class="QDialog" name="addemployeedialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>482</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox_2">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>260</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>0</y>
     <width>181</width>
     <height>251</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QSpinBox" name="idBox">
      <property name="maximum">
       <number>999999</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="nameEdit"/>
    </item>
    <item>
     <widget class="QLineEdit" name="surnameEdit"/>
    </item>
    <item>
     <widget class="QSpinBox" name="salaryBox">
      <property name="maximum">
       <number>999999</number>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="isManager">
      <property name="text">
       <string>Добавить менеджера</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="departmentEdit"/>
    </item>
    <item>
     <widget class="QSpinBox" name="empNumBox"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>0</y>
     <width>231</width>
     <height>141</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="idLabel">
      <property name="text">
       <string>Ввндите ID нового сотрудника:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="nameLabel">
      <property name="text">
       <string>Введите имя нового сотрудника:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="surnameLabel">
      <property name="text">
       <string>Введите фамилию нового сотрудника:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="salaryLabel">
      <property name="text">
       <string>Введите зарплату нового сотрудника:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>170</y>
     <width>277</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="departmentLabel">
      <property name="text">
       <string>Название отедела, в котором работает менеджер:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="empNumLabel">
      <property name="text">
       <string>Количество подчинённых менеджера:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
